setup.py

```diff
@@ -119,7 +119,7 @@ def get_git_sha():
         "postgres": ["psycopg2-binary==2.8.5"],
         "presto": ["pyhive[presto]>=0.4.0"],
         "elasticsearch": ["elasticsearch-dbapi>=0.1.0, <0.2.0"],
-        "druid": ["pydruid==0.6.0", "requests==2.22.0"],
+        "druid": ["pydruid>=0.6.1,<0.7"],
```

Comment:
The explicit dependency on `requests` is redundant, as it's already defined in `pydruid`.